pkg/client/restclient/url_utils.go

```diff
@@ -64,30 +68,28 @@ func DefaultServerURL(host, apiPath string, groupVersion unversioned.GroupVersio
 	// Add the version to the end of the path
 	if len(groupVersion.Group) > 0 {
 		versionedAPIPath = path.Join(versionedAPIPath, groupVersion.Group, groupVersion.Version)
-
 	} else {
 		versionedAPIPath = path.Join(versionedAPIPath, groupVersion.Version)
-
 	}
 
-	return hostURL, versionedAPIPath, nil
+	return hostURLs, versionedAPIPath, nil
```

Comment:
This seems like it would be better as a separate method that is DefaultServerURLs that calls DefaultServerURL